images/win/scripts/Installers/Install-PowerShellModules.ps1

```diff
@@ -0,0 +1,39 @@
+################################################################################
+##  File:  Install-PowerShellModules.ps1
+##  Desc:  Install PowerShell Modules.
+################################################################################
+
+$ErrorActionPreference = "Stop"
+
+# Set TLS1.2
+[Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor "Tls12"
+
+Write-Host "Setup PowerShellGet"
+Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+
+# Specifies the installation policy
+Set-PSRepository -InstallationPolicy Trusted -Name PSGallery
+
+# Install PowerShell modules
+$modules = (Get-ToolsetContent).powershellModules
+
+foreach($module in $modules)
+{
+    $moduleName = $module.name
+    Write-Host "Installing ${moduleName} module"
+
+    if ($module.versions)
+    {
+        foreach ($version in $module.versions)
+        {
+            Write-Host " - $version"
+            Install-Module -Name $moduleName -RequiredVersion $version -Scope AllUsers -SkipPublisherCheck -Force

```

Comment:
Just for curiosity: what version will be imported by default? Latest installed?

Reply:
PowerShell imports the newest version of the module that it finds in the first directory in which it finds the module